lib/tasks/encrypted_key.rake

```diff
@@ -2,16 +2,16 @@ namespace :rmt do
   namespace :secrets do
     desc 'Create encryption key for Rails secrets'
     task create_encryption_key: :environment do
-      require "rails/generators/rails/encryption_key_file/encryption_key_file_generator"
+      require 'rails/generators/rails/encryption_key_file/encryption_key_file_generator'
```

Comment:
As good as it is to remove rubocop offenses and have the code in a good state, this should be in a different PR.

Reply:
Sure! Usually our pipeline don't pass when there are Rubocop offenses, I really don't know how this one got into our main branch.

Reply:
Another PR merged today implemented the same changes.